engines/scumm/he/moonbase/ai_main.cpp

```diff
@@ -378,8 +380,8 @@ int AI::masterControlProgram(const int paramCount, const int32 *params) {
 		if (!getCoordinateVisibility(targetX, targetY, currentPlayer)) {
 			int closestHub = getClosestUnit(targetX, targetY, getMaxX(), currentPlayer, 1, BUILDING_MAIN_BASE, 1, 0);
 			int targetAngle = calcAngle(getHubX(closestHub), getHubY(closestHub), targetX, targetY);
-			int testX = static_cast<int>(getHubX(closestHub) + (500 * cos(degToRad(targetAngle))) + getMaxX()) % getMaxX();
-			int testY = static_cast<int>(getHubY(closestHub) + (500 * sin(degToRad(targetAngle))) + getMaxY()) % getMaxY();
+			int testX = static_cast<int>(getHubX(closestHub) + (500 * cos(Common::deg2rad<float>(targetAngle))) + getMaxX()) % getMaxX();
```

Comment:
This doesn't change anything in the end, but for consistency you might want to use `Common::deg2rad<int,float>()` here and on the line below since this is what you are doing in the rest of the file.